packages/react-native/examples/expo-app/package.json

```diff
@@ -2,17 +2,18 @@
   "name": "expo-app",
   "version": "1.0.0",
   "scripts": {
-    "start": "expo start --dev-client",
     "android": "npx expo run:android",
     "ios": "npx expo run:ios",
-    "web": "expo start --web"
+    "android:release": "npx expo run:android --variant release",
+    "ios:release": "npx expo run:ios --configuration Release",
+    "start": "expo start --dev-client"
   },
   "dependencies": {
-    "@honeybadger-io/react-native": "^5.1.1",
-    "expo": "~47.0.12",
+    "@honeybadger-io/react-native": "file:../../honeybadger-io-react-native-5.1.6_517150.tgz",
```

Comment:
I'm fine with just changing this to `"@honeybadger-io/react-native": "latest"`, however I do think my testing procedure is not super obvious and should be documented. Before I add documentation, can you think of a better way to do it @subzero10?  

Here's what I've been doing: 

1. In the `react-native` folder, run `npm run build && npm pack`
2. Rename the tarball to append some unique string and add the tarball file as the dependency in the test project(s) I'm working on (for a while I struggled because changes to the package were not coming through and it turned out that there was some caching issue. Renaming the file fixes that, however it's quite annoying)
3. `npm install` 

I tried just installing the package without making a tarball first like `"@honeybadger-io/react-native": "file:../.."`, but that never worked for me.

Reply:
Actually, I didn't know about this trick before seeing your approach with the tarball file!

I've had the same issue when installing with `"file:../.."` so what I would usually do is install the from npm (`"@honeybadger-io/react-native": "latest"`) and then modify code in `node_modules` while developing. So yeah, I don't have a good answer 😓.

Reply:
Ok, I think I will go with "latest" and add a note in the README.